src/analysis/properties.rs

```diff
@@ -199,7 +217,64 @@ fn analyze_property(
         None
     };
 
-    (getter, setter)
+    let mut used_types: Vec<String> = Vec::with_capacity(4);
+    let trampoline_name = trampolines::analyze(
+        env,
+        &library::Signal {
+            name: format!("notify::{}", name),
+            parameters: Vec::new(),
```

Comment:
Somehow we need to tell it to generate a `gpointer` parameter for the `extern "C"` trampoline but ignore it for anything else except for just having it there.